examples/random-login/consumer/Pulumi.yaml

```diff
@@ -11,8 +11,6 @@ resources:
     type: random-login:RandomLogin
     properties:
       petName: true
-      passwordLength:
-        fn::secret: 8
```

Comment:
Removing the `passwordLength` input property of `RandomLogin` because the component now generates a variable-length password,.